doc/data/messages/n/no-self-use/details.rst

```diff
@@ -1 +0,0 @@
-You can help us make the doc better `by contributing <https://github.com/PyCQA/pylint/issues/5953>`_ !
```

Comment:
Makes perfect sense. Perhaps we could use your Parisian example since that demonstrates both separate-function & staticmethod alternatives.

Should we also update [this description](https://github.com/PyCQA/pylint/blob/main/pylint/extensions/no_self_use.py#L33) to say: "could be written as a function **or staticmethod**"?
There is already a [functional test](https://github.com/PyCQA/pylint/blob/main/tests/functional/ext/no_self_use/no_self_use.py#L152) covering the staticmethod scenario